docs/sql-reference/sql-functions/math-functions/width_bucket.md

```diff
@@ -0,0 +1,42 @@
+# width_bucket
+
+## Description
+
+Returns the bucket number to which value would be assigned in an equiwidth histogram with num_bucket buckets, in the range min_value to max_value.
+
+## Syntax
+
+```Shell
+WIDTH_BUCKET( <expr> , <min_value> , <max_value> , <num_buckets> )
```

Comment:
Can we add an limitation that `min_value`, `max_value` and `num_buckets` should only be literal?